### PR TITLE
WRO-2580: Fixed `Scroller` to set initial focus to scrollbar in Panels

### PR DESCRIPTION
**Pull request by jeonghee27**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
 Initial focus is not set on the Scroller

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Add ScrollbarPlaceholder node that delegates focus to other nodes.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-19179 

### Comments
Enact-DCO-1.0-Signed-off-by: Jeonghee Ahn (jeonghee27.ahn@lge.com)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 ### Fixed
 
 - `sandstone/Scroller` and `sandstone/VirtualList` to focus the topmost element after scroll in pointer mode
+- `sandstone/Scroller` to give the scroll thumb the initial focus
```
Comment (MikyungKim) [inline CHANGELOG.md:17]: Please put it in the right place.

File: useScroll/ScrollbarPlaceHolder.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+/**
+ * ScrollbarPlaceHolder component.
+ *
+ * @class ScrollbarPlaceHolder
+ * @memberof sandstone/useScroll
+ * @ui
+ * @private
+ */
+const ScrollbarPlaceHolder = () => {
```
Comment (MikyungKim) [inline useScroll/ScrollbarPlaceHolder.js:15]: Let's rename this to `ScrollbarPlaceholder`.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact sandstone module, newest changes on the top.
 
+## [unreleased]
+
+### Fixed
+
+- `sandstone/Scroller` to give the scroll thumb the initial focus
```
Comment (MikyungKim) [inline CHANGELOG.md:9]:
```suggestion
- `sandstone/Scroller` to focus scroll thumb initially when it is used in Panels
```

**Comment by MikyungKim** [comment]
Please update the branch with the latest develop.

File: useScroll/ScrollbarPlaceHolder.js (hunk 2 above)
Comment (jeonghee27) [reply useScroll/ScrollbarPlaceHolder.js]: I fixed

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact sandstone module, newest changes on the top.
 
+## [unreleased]
+
+### Fixed
+
+- `sandstone/Scroller` focus scroll thumb initially when it is used in Panels
```
Comment (MikyungKim) [inline CHANGELOG.md:9]:
```suggestion
- `sandstone/Scroller` to focus scroll thumb initially when it is used in Panels
```

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**